crates/apub/src/protocol/objects/page.rs

```diff
@@ -46,7 +46,7 @@ pub struct Page {
   pub(crate) attributed_to: AttributedTo,
   #[serde(deserialize_with = "deserialize_one_or_many")]
   pub(crate) to: Vec<Url>,
-  pub(crate) name: String,
+  pub(crate) name: Option<String>,
```

Comment:
I've confirmed that this line is the one causing the test failures.

Reply:
Thats the main change in this PR. If we remove the option, then posts from Mastodon wont work.